std-bits/table/src/main/java/org/enso/table/data/column/operation/unary/TextCharLengthOperation.java

```diff
@@ -0,0 +1,53 @@
+package org.enso.table.data.column.operation.unary;
+
+import org.enso.base.Text_Utils;
+import org.enso.table.data.column.builder.Builder;
+import org.enso.table.data.column.operation.StorageIterators;
+import org.enso.table.data.column.operation.UnaryOperation;
+import org.enso.table.data.column.storage.ColumnStorage;
+import org.enso.table.data.column.storage.type.IntegerType;
+import org.enso.table.data.column.storage.type.TextType;
+import org.enso.table.data.table.problems.MapOperationProblemAggregator;
+
+public class TextCharLengthOperation implements UnaryOperation {
```

Comment:
No issue with this but we could do it as a different mode on `TextLengthOperation`?